kartothek/io/dask/_update.py

```diff
@@ -51,14 +87,16 @@ def _update_dask_partitions_shuffle(
     if ddf.npartitions == 0:
         return ddf
 
+    group_cols = partition_on[:]
```

Comment:
is this an implicit copy? If so, I would prefer `copy.copy` instead for readability/clarity.

Reply:
sure, can do